client/app/components/groupInfo/groupInfo.html

```diff
@@ -2,7 +2,7 @@
   <md-toolbar class="md-hue-1">
     <div class="md-toolbar-tools">
       <h1 class="md-title" flex>
-        <md-button ui-sref="groupList" class="md-icon-button">
+        <md-button ui-sref="joinGroupList" class="md-icon-button">
```

Comment:
This seems to be the rough steps to get it running:

1. setup and run the frontend dev server
2. in a new terminal, switch to the `foodsaving-e2e` repo
3. run `npm install`
4. update webdriver
```
$(npm bin)/webdriver-manager update
$(npm bin)/webdriver-manager start
```
5. in yet another terminal, switch to foodsaving-e2e and run `npm test`

this should open chrome and report something like this in the terminal:

```

> foodsaving-e2e@1.0.0 test /home/tic/Development/foodsaving-e2e
> protractor

(node:31929) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
[16:30:26] I/launcher - Running 1 instances of WebDriver
[16:30:26] I/hosted - Using the selenium server at http://localhost:4444/wd/hub

  ✓ e2e suite landing page should have a title: 1346ms
  ✓ e2e suite landing page should have a login button: 1333ms

  2 passing (5s)

[16:30:33] I/launcher - 0 instance(s) of WebDriver still running
[16:30:33] I/launcher - chrome #01 passed
```